api/SharedWorkerGlobalScope.json

```diff
@@ -0,0 +1,287 @@
+{
+  "api": {
+    "SharedWorkerGlobalScope": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SharedWorkerGlobalScope",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "29"
+          },
+          "firefox_android": {
+            "version_added": "4"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "10.6"
+          },
+          "opera_android": {
+            "version_added": "11.5"
+          },
+          "safari": {
+            "version_added": "5"
+          },
+          "safari_ios": {
+            "version_added": "7.1"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "open": {
```

Comment:
I don't understand what `open` is or where it comes from. @chrisdavidmills you've added this to https://developer.mozilla.org/en-US/docs/Web/API/SharedWorkerGlobalScope#Browser_compatibility, what is it?

Reply:
@Elchi3 this is an interesting mystery — I can't find any record of this ```open``` property anywhere, and running ```console.log(self.open)``` inside a shared worker returns ```undefined```. I wonder if maybe it was added at some point, but then removed from the spec again? @jpmedley, any thoughts on this?

Reply:
It's a mystery to me too.

Reply:
Let's remove this then please.

Reply:
Yup, agreed.

Reply:
OK. Removed :)